QueryDemo.ipynb

```diff
@@ -0,0 +1,3670 @@
+       " ('2MASS J03510004-0052452', 57.750083, -0.87940833, None, None, '0351-0052', 'Luyt79.5', None, None),\n",
+       " ('CWISE J153143.38-330657.3', 232.93075, -33.11591666666667, None, None, None, 'Kirk21', None, None),\n",
+       " ('2MASS J17312974+2721233', 262.873917, 27.356472, None, None, '1731+2721', 'Reid08.1290', None, None),\n",
+       " ('PSO J318.4243+35.1277', 318.4239, 35.128, None, '2000', None, 'Alle16', None, None),\n",
+       " ('ULAS J232600.40+020139.2', 351.5018, 2.0273, None, '2000', None, 'Burn13', None, None),\n",
+       " ('TrES-3', 268.029, 37.5462, 2000.0, 'J2000', None, 'ODon07', None, None),\n",
+       " ('ULAS J013017.79+080453.9', 22.5742, 8.0814, None, '2000', None, 'Burn13', None, None),\n",
+       " ('2MASS J23494899+1224386', 357.4542, 12.4107, None, '2000', None, 'Gizi00', None, None),\n",
+       " ('SDSS J122711.07+021349.1', 186.7961, 2.2302, None, '2000', None, 'West08', None, None),\n",
+       " ('SDSS J140409.91+400214.0', 211.0411, 40.0372, None, '2000', None, 'West08', None, None),\n",
+       " ('2MASS J10595185+3042059', 164.96604, 30.701639, None, None, '1059+3042', 'Shep09', None, None),\n",
+       " ('SDSS J172545.59+640501.2', 261.4397, 64.0836, None, '2000', None, 'Kell15', None, None),\n",
+       " ('WISE J190624.74+450807.1', 286.603125, 45.135611, None, None, '1906+4508', 'Kirk11', None, None),\n",
+       " ('Hya08', 74.6903, 12.2095, None, '2000', None, 'Lodi14.3908', None, None),\n",
+       " ('CWISE J013719.04+342837.6', 24.3293435, 34.4771182, 2015.4041, 'ICRS', None, 'Roth', None, None),\n",
+       " ('2MASS J11183877+2339488', 169.66155, 23.663563, None, None, None, 'Cruz03', None, None),\n",
+       " ('CFHT-IC 348-18', 56.2055, 32.1637, None, '2000', None, 'Alve13.123', None, None),\n",
+       " ('SSSPM J0306-3648', 46.5483, -36.798, None, '2000', None, 'Lodi05', None, None),\n",
+       " ('CWISEP J225628.97+400227.3', 344.12070833333325, 40.04091666666667, None, None, None, 'Meis20.74', None, None),\n",
+       " ('ULAS J094349.60+094203.4', 145.9566, 9.7012, None, '2000', None, 'Burn10.1885', None, None),\n",
+       " ('SDSS J144635.65+164337.3', 221.6485, 16.7271, None, '2000', None, 'Schm10.1808', None, None),\n",
+       " ('2MASS J14345819-2335572', 218.7423, -23.5993, None, '2000', None, 'Ruiz01', None, None),\n",
+       " ('2MASS J03550477-1032415', 58.76989, -10.544882, None, None, '0355-1032', 'Cruz03', None, None),\n",
+       " ('2MASS J06022216+6336391', 90.5923335, 63.610861, None, None, '0602+6336', 'Reid08.1290', None, None),\n",
+       " ('WISEA J084254.56-061023.7', 130.7285, -6.1731, None, '2000', None, 'Grec19', None, None),\n",
+       " ('2MASS J00285545-1927165', 7.2310417, -19.454583, None, None, '0028-1927', 'Reid08.1290', None, None),\n",
+       " ...]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "db.query(db.Sources).all()    "
```

Comment:
To avoid so many results consider adding a limit, may also want to output as a table for easier viewing:
`db.query(db.Sources).limit(10).table()`

Even if you don't apply a limit, I think the notebook's rendering of astropy tables will be better than the default tuple output.